aws/ebs.go

```diff
@@ -43,6 +43,17 @@ func getAllEbsVolumes(session *session.Session, region string, excludeAfter time
 	return volumeIds, nil
 }
 
+// hasEBSExcludeTag checks whether the exlude tag is set for a resource to skip deleting it.
+func hasEBSExcludeTag(volume *ec2.Volume) bool {
+	// Exclude deletion of any buckets with cloud-nuke-excluded tags
+	for _, tag := range volume.Tags {
+		if *tag.Key == AwsResourceExclusionTagKey && *tag.Value == "true" {
```

Comment:
NIT - In general we want to use the `aws.StringValue` and related methods for dereferencing pointers - partly for consistency, but mainly for the fact that these methods include a safety check to ensure we're not dereferencing a nil pointer.